promql/engine.go

```diff
@@ -752,6 +752,7 @@ func (ng *Engine) execEvalStmt(ctx context.Context, query *query, s *parser.Eval
 		case parser.ValueTypeScalar:
 			return Scalar{V: mat[0].Floats[0].F, T: start}, warnings, nil
 		case parser.ValueTypeMatrix:
+			sort.Sort(mat)
```

Comment:
Let's also add the `sortSpanTimer` from the range query to be consistent.

Reply:
Good catch, added in Good catch, added in 0e934db.